numpy/core/src/multiarray/datetime.c

```diff
@@ -3904,7 +3904,8 @@ datetime_to_timedelta_resolve_descriptors(
         PyArrayMethodObject *NPY_UNUSED(self),
         PyArray_DTypeMeta *dtypes[2],
         PyArray_Descr *given_descrs[2],
-        PyArray_Descr *loop_descrs[2])
+        PyArray_Descr *loop_descrs[2],
+        npy_intp *NPY_UNUSED(view_offset))
```

Comment:
Is this change intentional?

Reply:
The change is correct, but was split out into its seperate PR already.